src/js/events.js

```diff
@@ -42,8 +42,13 @@ EventEmitter.prototype.emit = function(type) {
 
   var listeners = this._events[type];
   if (util.isArray(listeners)) {
-    listeners = listeners.slice();
-    var args = Array.prototype.slice.call(arguments, 1);
+    var args;
+    switch (arguments.length) {
+      case 0: args = []; break;
```

Comment:
Shall switch `arguments.length` starts on 1?

Reply:
Then how do we handle the case 0?

Reply:
Never mind, that's my bug 👍